src/redux/services/schema.ts

```diff
@@ -7,8 +7,11 @@ import {isKustomizationResource} from '@redux/services/kustomize';
 
 import {getResourceKindHandler} from '@src/kindhandlers';
 
+// @ts-ignore
 const k8sSchema = JSON.parse(loadResource('schemas/k8sschemas.json'));
```

Comment:
Should we make a function that does `JSON.parse` and `loadResource` in a try catch and use that instead? This would crash the app if the parsing fails

Reply:
in this specific case it was intentional to let the app crash if the this parsing fails - as it is required to run correctly - but perhaps we can handle the situation more gracefully..

Reply:
Right, then we could leave it like this or we could show an error alert saying that some of the features failed to load. But I'm not sure if we assume that this works/exists in other places which might crash the app later if we don't crash it to start with, hmm..